src/app/app-routing.module.ts

```diff
@@ -9,12 +9,12 @@ import { HomeComponent } from './home/home.component';
 const routes: Routes = [
   {
     path: 'users',
-    canActivate: [RouteGuard],
+    // canActivate: [RouteGuard],
```

Comment:
Why is this commented?

Reply:
With out login we cannot go inside just to differentiate with platform_admin and organisation_admin